modules/generator/processor/spanmetrics/config.go

```diff
@@ -27,6 +29,9 @@ type Config struct {
 	// Additional dimensions (labels) to be added to the metric. The dimensions are generated
 	// from span attributes and are created along with the intrinsic dimensions.
 	Dimensions []string `yaml:"dimensions"`
+	// Subprocessor options for this Processor include Latency, Counts
+	// These are metrics categories that exist under the umbrella of Span Metrics
+	Subprocessors map[string]bool
```

Comment:
I'm wondering if this could be `map[SubProcessor]bool` to force us to be specific about the key space.

Reply:
@zalegrala Sure, I could make type ``Subprocessor`` with field ``Name``. If we wanted to enforce specificity about the keys e.g. Latency and Count we could make a constructor for the type that does a check to make sure the value matches one of the choices.

Reply:
Sure, a struct or an int iota sort of approach could probably work also.